public/login.js

```diff
@@ -99,15 +92,32 @@ export function login() {
     }
   });
 
-return (
-  <>
-      <h1 id="loggedInStatus">You are not yet logged in</h1>
-      <input id="email" type="email" placeholder="Email"/><br/>
-      <input id="password" type="password" placeholder="Password"/><br/><br/>
-      <button id="login">Login</button>
-      <button id="signup">SignUp</button>
-      <button id="logout" style="display:none;">Logout</button>
-      <button id="googlelogin">Google Login</button>
-  </>
-);
-}
\ No newline at end of file
+  return (
+    <>
+      <h1 id='loggedInStatus'>You are not yet logged in</h1>
+      <input id='email' type='email' placeholder='Email' />
+      <br />
+      <input id='password' type='password' placeholder='Password' />
+      <br />
+      <br />
+      <button
+        id='login'
+        onClick={(e) => {
+          const auth = firebase.auth();
+          const promise = auth.signInWithEmailAndPassword(
+            email.value,
+            password.value
+          );
+          promise.catch((e) => console.log(e.message));
+        }}
+      >
+        Login
+      </button>
+      <button id='signup'>SignUp</button>
+      <button id='logout' style={{ display: "none" }}>

```

Comment:
This was also crashing. Since it's a jsx component you need to provide inline style properties inside an object.